src/components/Map/MapView.tsx

```diff
@@ -186,6 +186,11 @@ export default function MapView(props: IProps) {
   const onMouseEnter = React.useCallback(() => setCursor("pointer"), []);
   const onMouseLeave = React.useCallback(() => setCursor("auto"), []);
 
+  // biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
```

Comment:
todo (blocking): Could you add an explanation here – and/or a short comment why we need this?